modules/claims_api/config/schemas/v2/power_of_attorney_requests/param/decision/post.json

```diff
@@ -10,7 +10,6 @@
       "type": "object",
       "additionalProperties": false,
       "required": [
-        "type",
```

Comment:
Not related to this task, but realized we aren't using this `type` in the decision docs.